File: examples/Person.py
class Person:
    # Metodos Publicos
    # Constructor
    def __init__(self, name, age):
        self.name = name
        self.age = age
    # Getters
    def getName(self):
        return self.name
    def getAge(self):
        return self.age
    # Setters
    def setName(self, name):
        self.name = name
    def setAge(self, age):
        self.age = age
# Instancia
user = Person("cristian", 18)
user.setAge(33)

File: examples/Person.php
<?php 
class Person {
    // Propiedades Privadas
    private $nombre;
    private $apellido;
    private $edad;
    // Metodo Constructor
    public function __construct($name, $age) {
        $this->name = $name;
        $this->age = $age;
    }
    // Getters
    public function getName(){
        return $this->name;
    }
    public function getAge(){
        return $this->age;
    }
    // Setters
    public function setName($name){
        $this->name = $name;
    }
    public function setAge($age){
        $this->age = $age;
    }
}
// Instancia
$user = new Person("cristian", 18);
$user->setAge(33);
?>